main.py

```diff
@@ -1,7 +1,24 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
-from django.db.models import QuerySet
+def main() -> None:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the code, but it's better to avoid creating incorrect entries initially.